internal/handlers/user.go

```
@@ -32,6 +38,41 @@ func (r *userHandler) Get(ginCtx *gin.Context) {
 
 	ginCtx.JSON(http.StatusOK, userResponse)
 }
+func (r *userHandler) Update(ginCtx *gin.Context) {
+	var userRequest dto.UserPatchRequest
+
+	userID := ginCtx.Param("id")
+
+	userIDInt, _ := strconv.Atoi(userID)
```

don't ignore error here, return bad request if not valid id

---

done ya